src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,75 @@
+import { Platform } from 'react-native';
+
+import { secrets } from '../config';
+import * as SecureStore from 'expo-secure-store';
+import * as appJson from '../../app.json';
+
+
+const namespace = appJson.expo.slug as keyof typeof secrets;
```

Comment:
you can import `namespace` directly from `configs`. or does that not work because of needing types?

Reply:
exactly.
if we import it from configs it is just a `string` value, which causes problems when accessing the secrets object with it as a key.

should we change the namespace file to be a ts file now, or keep it as is and do it sometime in the future?

Reply:
all right. lets do it next time we need namespace in ts.